src/medUtilities/medUtilities.h

```diff
@@ -40,6 +40,13 @@ class MEDUTILITIES_EXPORT medUtilities
     static medAbstractData *changeMaxNumberOfMeshTriangles(medAbstractData *mesh, int value);
 
     static double minimumValue(dtkSmartPointer<medAbstractData> data);
+
+    /**
+     * @brief maximumValue computes the maximum pixel intensity in a volume
+     * @param the input volume
+     * @return the maximum intensity value (double)
+     */
+    static double maximumValue(dtkSmartPointer<medAbstractData> data);
```

Comment:
Could you just place a similar comment section for ```minimumValue``` for homogeneity please ?

Reply:
Done